### PR TITLE
chore: Add hotfix CI pipeline, remove archive pipeline

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
part of #3727
#### Please provide a brief summary of this pull request.
This is safe version with commented crucial parts of CI (publishing/creating release notes). Just to validate pipeline

Also CI for archive-branch has been removed - it is outdated

### CODE REVIEW

File: ci-scripts/hotfix-publish.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -u -e
+
+git config --global user.email "fundamental@sap.com"
```
Comment (Betrozov) [inline ci-scripts/hotfix-publish.sh:5]: @JKMarkowski How about using an environment variable here?

**Review by Betrozov — CHANGES_REQUESTED** [review]

File: ci-scripts/hotfix-publish.sh (hunk 1 above)
Comment (JKMarkowski) [reply ci-scripts/hotfix-publish.sh]: @Betrozov sure, done  could you validate it? I'm not very into sh scripts

Comment (Betrozov) [reply ci-scripts/hotfix-publish.sh]: @JKMarkowski I'm talking about to use the Travis environment variable :slightly_smiling_face: 
and for it we not need to create `.ci-env/github.sh` we can just to user environment variable that will be located on Travis

Comment (JKMarkowski) [reply ci-scripts/hotfix-publish.sh]: Ok, It makes more sense :laughing: 
image

**Review by Betrozov — APPROVED** [review]

File: ci-scripts/hotfix-publish.sh (hunk 1 above)
Comment (Betrozov) [reply ci-scripts/hotfix-publish.sh]: Great job 👍
Thanks 🙂

**Review by InnaAtanasova — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**